Add basic functionality to Discord bot

AlexisMLopez opened the pull request:
Add several basic functions to the Discord bot, including the ability to use slash commands.

Commit:
Add slash command ability to Discord bot
Convert previous commands to slash commands
Add 8ball command
Add Hello command

RonnyFrayRegato commented on discord-script.py:
```diff
@@ -1,19 +1,67 @@
+    # bot syncs slash commands on startup
+    async def on_ready(self):
+        await tree.sync(guild=discord.Object(
+            id=1030163730538954853))  # without this the slash commands will not instantly update (encrypt later)
```
The id parameter's value should be converted into a variable since it's used several times throughout the file.

AlexisMLopez replied:
The server ID variable has been added.

RonnyFrayRegato commented on discord-script.py:
```diff
@@ -1,19 +1,67 @@
+    # bot syncs slash commands on startup
+    async def on_ready(self):
+        await tree.sync(guild=discord.Object(
+            id=1030163730538954853))  # without this the slash commands will not instantly update (encrypt later)
```
Does the Discord server ID need to be encrypted for security?

AlexisMLopez replied:
It does not because server IDs are public to anyone that owns a Discord account.

Commit:
Add slash command ability to Discord bot
Convert previous commands to slash commands
Add 8ball command
Add Hello command
Create server ID variable
Remove security TODO

Commit:
Create server ID variable
Remove security TODO

Commit:
Create server ID variable
Remove security TODO
Remove Discord token

Commit:
Merge remote-tracking branch 'origin/add-basic-functionality' into add-basic-functionality

Commit:
Remove Discord token

zachary-mixson reviewed (approved):
looks good!
